integration-tests/showSolutionListPage.test.js

```diff
@@ -97,3 +97,14 @@ test('should navigate to the solution view page when clicking on the title of th
     .expect(getLocation())
     .contains('/view-solution/S1');
 });
+
+test('should navigate to browse solutions page when click Go back', async (t) => {
+  pageSetup(t);
+
+  const goBackButton = Selector('[data-test-id="go-back-button"] a');
+
+  await t
+    .expect(goBackButton.exists).ok()
+    .click(goBackButton)
+    .navigateTo('./browse-solutions');
```

Comment:
Does navigate actually check the content of the URL? We switched to use contains because navigateTo didn't seem to work?